pkg/services/app/add.go

```diff
@@ -581,10 +581,10 @@ func makeWegoApplication(params AddParams) wego.Application {
 	return app
 }
 
-func generateAppYaml(params AddParams, repo string) ([]byte, error) {
+func generateAppYaml(params AddParams) ([]byte, error) {
 	app := makeWegoApplication(params)
 
-	appHash, err := utils.GetAppHash(repo, params.Path, params.Branch)
+	appHash, err := utils.GetAppHash(params.Url, params.Path, params.Branch)
```

Comment:
If you change it here you need to also change where we are checking if the appHash exists already when adding a new app.